cloudflare.php

```diff
@@ -502,3 +502,7 @@ function theme_save_pressed()
 // Enable HTTP2 Server Push
 require_once plugin_dir_path(__FILE__).'src/HTTP2ServerPush.php';
 add_action('init', array('HTTP2ServerPush', 'init'));
+
+// Display error message if user is using a subdomain
+require_once plugin_dir_path(__FILE__).'src/SubdomainChecker.php';
```

Comment:
What I meant was Composer's auto loader should include everything in src/ so you should be able to do `vendor/autoload.php` at the top of the file and then remove the requires for SubdomainChecker.php and HTTP2ServerPush.php